src/addBusinessDays/index.ts

```diff
@@ -25,40 +24,59 @@ import isSaturday from '../isSaturday/index'
  */
 export default function addBusinessDays(
   dirtyDate: Date | number,
-  dirtyAmount: number
+  dirtyAmount: number,
+  dirtyOptions?: {
+    businessDays?: number[]
+  }
```

Comment:
Maybe I'm not entirely following how the options work, but I'm not sure the array of weekdays is the most DX-friendly way to go to exclude holidays next to weekends, but it could work as one version of doing so.

@kossnocorp Would an api with an array of holidays as a second way to exclude them be a good idea?

e.g.

```suggestion
  dirtyOptions?: {
    businessDays?: number[],
    excludeDates?: (Date | number)[],
  }
```

Reply:
Thanks for your feedback and suggestion!
Having excludeDates would be nice, but for parameter type `Date[]` makes it clearer when to use it imo.
So businessDays, I was thinking of a case when business days are not from Monday to Friday (e.g. A store is open only on Monday, Wednesday, Friday, then businessDays would be `[1, 3, 5]` ).
But excludeDates (or holidays?) wants to solve the cases like excluding certain dates, whereas businessDays is for days of the week.

Reply:
Right, that makes tot sense 🙃 Had not considered that scenario. It is a good api for solving that specific usecase. I still have to get around to testing the code, but I’m definitely a fan of the api now.

(I think I’ll try to add the holidays solution on dirtyOptions once this is merged, but I’m hesitant to sign off on this without the docs building or one of the core contributors giving their approval ofcourse)

Reply:
> I think I’ll try to add the holidays solution on dirtyOptions once this is merged
cool 👍